1 q2025 refresh

JAMOGRAD opened the pull request:
An update to CICS Banking Sample Application.

Maintenance changes to the Java and JavaScript applications.
Maven Wrapper to build them all.
Improvements to JCL for Db2.
Documentation enhancements.

PhilWakelin commented:
1. The maven wrapper works for me, but I can't see any instructions in the main readme about using it to build the sample. Note I can see there are good instructions in the installation guides, so we could just link to them
2. There are probably two internal files packaged in the root dir
`.pre-commit-config.yaml` 
`.secrets.baseline`
3. The second installing link in the readme does not work as it refers to the `libertyUI` directory
```
The Carbon React UI installation, this is optional, requires a JVM server to be running in the CICS region (this gets set up as part of installation process). See [the Carbon React UI installation documentation.](https://github.com/cicsdev/cics-banking-sample-application-cbsa/tree/main/etc/install/libertyUI/doc/CBSA_Liberty_UI_installation_deployment_guide.md)
```
4. The [maintainers ](https://github.com/cicsdev/cics-banking-sample-application-cbsa/blob/1Q2025Refresh/MAINTAINERS.md)file date has not been updated.

JAMOGRAD commented:
> 1. The maven wrapper works for me, but I can't see any instructions in the main readme about using it to build the sample. Note I can see there are good instructions in the installation guides, so we could just link to them

I now say in the main README

`- A Maven wrapper is used to build the Java components. This is included for your convenience.`

and you don't have to download Maven any more (so I removed that line).

> 
>     2. There are probably two internal files packaged in the root dir
>        `.pre-commit-config.yaml`
>        `.secrets.baseline`

Initially I wasn't sure about these, so I looked on "public" Github to see if other people were checking them in. And it's quite a common file. These do seem to be a thing we *should* check in and don't seem to contain any secrets.
> 
>     3. The second installing link in the readme does not work as it refers to the `libertyUI` directory
> 
> 
> ```
> The Carbon React UI installation, this is optional, requires a JVM server to be running in the CICS region (this gets set up as part of installation process). See [the Carbon React UI installation documentation.](https://github.com/cicsdev/cics-banking-sample-application-cbsa/tree/main/etc/install/libertyUI/doc/CBSA_Liberty_UI_installation_deployment_guide.md)
> ```
> 

Good spot! Fixed that!

>     4. The [maintainers ](https://github.com/cicsdev/cics-banking-sample-application-cbsa/blob/1Q2025Refresh/MAINTAINERS.md)file date has not been updated.

It has now! Thanks for that.
